kpm/lib/kpm/base_artifact.rb

```diff
@@ -273,6 +274,22 @@ def path_looks_like_a_directory(path)
         # Probably a directory
         true
       end
+
+      def remove_old_default_bundles(coordinate_map,artifact_info)
+        return unless coordinate_map[:artifact_id] == 'killbill-platform-osgi-bundles-defaultbundles'
+
+        default_bundles = Dir.glob("#{artifact_info[:dir_name]}/killbill-platform-osgi-bundles-*.jar")
```

Comment:
Unfortunately, this is not as easy, as this will miss other, non-Kill Bill, bundles (e.g. `org.apache.felix.bundlerepository-1.6.6.jar`).

It would be better to look at what the `tgz` would install (`foo-x.y.z.jar`, `bar-x.y.z.jar`, ...) and remove other versions of `foo` and `bar` on the filesystem.